src/commands/react_native/appcenter.rs

```diff
@@ -134,7 +134,10 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
     let mut processor = SourceMapProcessor::new();
 
     for path in matches.values_of("paths").unwrap() {
-        for entry in (fs::read_dir(path)?).flatten() {
+        let entries = fs::read_dir(path)
+            .map_err(|e| anyhow!(e).context(format!("Failed processing path: \"{}\"", &path)))?;
```

Comment:
there is `with_context` for this: https://docs.rs/anyhow/latest/anyhow/trait.Context.html#example